circle.yml

```diff
@@ -516,7 +516,7 @@ commands:
     - run:
         name: Check if Firebase should be skipped
         command: |
-          SKIPPABLE_TAG=$( git log -1 | grep -ioE -e "\[(skip.firebase|firebase.skip)\]" -e "\[((i|mac)os)+(, (i|mac)os)?\]" -e "\[darwin\]" || true )
+          SKIPPABLE_TAG=$( git diff origin/master... --cumulative | grep -ivE -e"ios|macos|darwin" || true )
```

Comment:
Oh, one question - is there a way for us to reference a diff against the target branch instead of hardcoding `master`? I have a release branch scenario in mind.

Reply:
Great catch! I guess `$CIRCLE_TARGET_BRANCH` should fit